mars/scheduler/graph.py

```diff
@@ -799,6 +813,36 @@ def build_tensor_merge_graph(self, tensor_key):
 
         return serialize_graph(graph)
 
+    def tile_fetch_tensor(self, tensor):
```

Comment:
We can let the tile done by the GraphActor which owned the original tensor, it would be much simpler.